manual/Searching/Autocomplete.md

```diff
@@ -45,7 +45,7 @@ POST /autocomplete
 ```
 
 #### Options
-- `layouts`: A comma-separated string of keyboard layout codes to validate and check for spell correction. Available options: us, ru, ua, se, pt, no, it, gr, uk, fr, es, dk, de, ch, br, bg, be (more details [here](../Searching/Spell_correction.md#Options)). Default: all enabled
+- `layouts`: A comma-separated string of keyboard layout codes to validate and check for spell correction. Available options: us, ru, ua, se, pt, no, it, gr, uk, fr, es, dk, de, ch, br, bg, be (more details [here](../Searching/Spell_correction.md#Options)). At least two layouts must be specified due to the combination logic. Default: all enabled
```

Comment:
"Due to the combination logic" sounds vague. Let's clarify that otherwise it's not clear what layouts to attempt to convert from and to.